packages/components/src/index.html

```diff
@@ -25,12 +25,45 @@
   </style>
 </head>
 <body>
-  <h1>Components</h1>
-  <h3>Alert</h3>
-  <t-alert custom-class="alert--success" variant="success">Success</t-alert>
-  <h3>Button</h3>
-  <t-button variant="primary">Hello</t-button>
-  <h3>Card</h3>
-  <t-card>A title</t-card>
+<h1>Components</h1>
+<h3>Alert</h3>
+<button
+  id="show-alert-btn">show alert
+</button>
+<t-alert
```

Comment:
Please add examples for all major usecases the component can handle:
1. No timeout
2. Default timeout
3. Custom timeout